index.html

```diff
@@ -59,6 +59,171 @@ <h2>I'm a software <br> developer</h2>
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="img-holder">
+                        <img src="img/card4.png" alt="project4">
+                    </div>
+                    <div class="text-holder">
+                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavScript</li>
+                            <li>Html</li>
+                        </ul>
+                        <button class="project-button" type="button">
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="img-holder">
+                        <img src="img/Img Placeholder.png" alt="project5">
+                    </div>
+                    <div class="text-holder">
+                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavScript</li>
+                            <li>Html</li>
+                        </ul>
+                        <button class="project-button" type="button">
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div class="img-holder">
+                        <img src="img/Img Placeholder.png" alt="project6">
+                    </div>
+                    <div class="text-holder">
+                        <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                        <ul>
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavScript</li>
+                            <li>Html</li>
+                        </ul>
+                        <button class="project-button" type="button">
+                            <a href="#">See Project</a>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </section>  
+         <!-- About me section -->
+         <section class="aboutme-section">
+            <div class="about-me">
+                <img src="img/Mask Group.png" alt="bg-img">
+                <h3>About me</h3>
+                <p class="flex"> I can help you build a product, <br> feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    <br> don’t hestiate to contact me.</p>
```

Comment:
- [ ] Same as here, please do these few changes, remember, it is recommanded you stcik on the figma design as much as posible by using the same `color, padding, font,..` all properties as used in the orignal design please.
- [ ] All your cards backgrounds are diffrent from the original design, kindly make sure you modify everything.
<img width="672" alt="ouai2" src="https://user-images.githubusercontent.com/77843050/205413612-e51c1283-ec51-4055-b8b5-cde637475460.png">